.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ commands:
         default: 6
     steps:
       - browser-tools/install-firefox:
-          version: 78.9.0esr
+          version: 91.5.0esr
       - browser-tools/install-geckodriver:
           version: v0.29.1
```

Comment:
Unrelated (and possibly worth tracking separately in Jira), but do we want to bump Geckodriver as well (here and L80 below, possibly others).
Looks like v0.29.1 was circa Apr 9, 2021 and v0.32.0 was released 15 days ago; per https://github.com/mozilla/geckodriver/releases

Reply:
Upgraded to ESR 102.4.0 and Geckodriver 0.32.0